packages/editor/src/commands/RemoveObjectsCommand.ts

```diff
@@ -63,7 +63,7 @@ export default class RemoveObjectsCommand extends Command {
           if (!parent) throw new Error('Parent is not defined')
           this.oldParents.push(parent)
 
-          const before = tree.entityNodeMap.get(parent.children![parent.children!.indexOf(obj.entity) + 1])
+          const before = tree.entityNodeMap.get(parent.children![parent.children!.indexOf(obj.entity) - 1])
```

Comment:
Index of the before object is `+1` than current object's index. So the previous formula was correct one. Also I accidentally fixed this delete issue in my last PR 😜. Try pulling the lastest and see if this issue is already resolved or not.

Reply:
#5556 this is the PR in which I addressed this issue.